images/runtime/dotnetcore/8.0/base.bookworm.staging.Dockerfile

```diff
@@ -0,0 +1,67 @@
+# syntax=docker/dockerfile:1.3
+# the above line allows this dockerfile to use the secrets fucntionality
+
+# dotnet tools are currently available as part of SDK so we need to create them in an sdk image
+# and copy them to our final runtime image
+FROM mcr.microsoft.com/dotnet/sdk:8.0 AS tools-install
+RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-sos
+RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-trace
+RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-dump
+RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-counters
+RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-gcdump
+RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-monitor --version 6.*
+
+FROM mcr.microsoft.com/mirror/docker/library/debian:buster-slim
+ARG BUILD_DIR=/tmp/oryx/build
+ADD build ${BUILD_DIR}
+
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+        # .NET Core dependencies
+        libc6 \
+        libgcc1 \
+        libgssapi-krb5-2 \
+        libicu63 \
```

Comment:
Could you please confirm that you copied the Dockerfiles for each version from their corresponding prod version, and not just duplicating the .NET 6 one?
For example in .NET 8 libicu should be targeting `libicu72` (cf. https://github.com/microsoft/Oryx/blob/cde2209830994c33600eb9e6658d2eba3c480710/images/runtime/dotnetcore/8.0/base.bookworm.Dockerfile)

There might be other differences so we need to rely on the existing .NET 7/8/6 Dockerfiles for these staging ones.

Reply:
Thanks Dan for pointing this out.  I have copied dockerfiles from dotnet 7 and reused it. As, I am using these dockerfiles just for testing purposes, I did not pay attention to the details here. Was thinking to create the actual Dockerfiles for release purposes from their corresponding version's base.Dockerfile in the next PR (PR-2 to release the valid images created from private binaires).
Should I update these dockerfiles in these PR?

Reply:
OK thanks for clarifying. I believe it should be fine as we're mostly checking the presence of the images generated and not their content? Let's not lose time with re-triggering a new iteration in that case.

Reply:
We need to double-check the dependencies outlined here because some of them are specific to certain versions of Debian.

For example, `libicu63` is for buster, `libicu67` is for bullseye and `libicu72` is for bookworm.

Similarly, both buster and bullseye used `libssl1.1`, but bookworm uses `libssl3`.

I _think_ these are the only cases, but we'll need to check out Dockerfiles to be certain.